java/src/jmri/jmrit/logixng/expressions/ExpressionBundle.properties

```diff
@@ -101,6 +101,11 @@ EntryExit_DestinationPointsInUseEntryExitExpressionVeto = DestinationPoints is i
 False_Short           = Always false
 False_Long            = Always false
 
+FileAsFlag_Short      = File as flag
+FileAsFlag_Long       = File as flag. Filename: {0}. {1}
+FileAsFlag_Delete     = Delete file if it exists
```

Comment:
The keep/delete options only apply to the true state.  The "if it exists" is redundant.  Maybe "Delete or keep:" should be something like "When found" or maybe "When true".

Reply:
Done. I changed it to "When true". I have also removed "if it exists".